src/deluge/storage/flash_storage.cpp

```diff
@@ -200,6 +201,8 @@ GridDefaultActiveMode defaultGridActiveMode;
 
 SampleRepeatMode defaultSliceMode;
 
+bool defaultShortSliceMode = true;
```

Comment:
Maybe we should use a `SampleRepeatMode` here instead -- that gives maximum flexibility for essentially no cost in terms of code size or flash storage.

If you're feeling extra fancy, the default short and regular slice modes could be packed in to the same byte in the flash storage:

```cpp
static_assert(kNumRepeatModes < (1 << 4), "2 repeat modes are packed in to 1 byte");
buffer[167] = (util::to_underlying(defaultShortSliceMode) << 4) | util::to_underlying(defaultSliceMode);
```

And on the read side:
```cpp
uint8_t defaultSliceModeRaw = buffer[167] & 0xf;
uint8_t defaultShortSliceModeRaw = (buffer[167] >> 4) & 0xf;

if (defaultSliceModeRaw >= kNumRepeatModes) {
   defaultSliceMode = SampleRepeatMode::CUT;
} else {
  defaultSliceMode = static_cast<SampleRepeatMode>(defaultSliceModeRaw);
}

if (defaultShortSliceModeRaw >= kNumRepeatModes) {
   defaultShortSliceMode = SampleRepeatMode::ONCE;
} else {
  defaultShortSliceMode = static_cast<SampleRepeatMode>(defaultShortSliceModeRaw);
}
```

(make sure I got the settings right for matching the default behavior....)

Reply:
i really dont know what any of that means. but i will check tomorrow again.

Reply:
it sounds like a fancy and good solution, but i am not capable of putting this all in the right place. the other comment i could just accept. is this also possible to do here?

Reply:
it's a more complex change than can really fit in the framework of touchups to individual hunks (the diff sections) unfortunately. I can give implementation guidance or eventually get around to implementing it myself.

Reply:
Or is it ok to just leave it like this? Does it impact performance / mess with guidelines or anything?